R/ui-legacy.R

```diff
@@ -137,24 +148,20 @@ ui_warn <- function(x, .envir = parent.frame()) {
 }
 
 
-# Silence -----------------------------------------------------------------
-
-#' @rdname ui
-#' @param code Code to execute with usual UI output silenced.
-#' @export
-ui_silence <- function(code) {
-  withr::with_options(list(usethis.quiet = TRUE), code)
-}
```

Comment:
`ui_silence()` is a keeper, so it moved to `utils-ui.R`, home of the new UI functions.